collectors/pod.go

```diff
@@ -324,13 +338,22 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 				c.Name, nodeName)
 		}
 
+		if gpu, ok := req[v1.ResourceNvidiaGPU]; ok {
```

Comment:
According to the [doc](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/), 
```
GPUs can be specified in the limits section only
```
Request nvidia cpu will be none exist for now, request nvidia cpu metric seems useless. @WIZARD-CXY